src/Processors/QueryPlan/DistinctStep.cpp

```diff
@@ -91,7 +112,11 @@ void DistinctStep::transformPipeline(QueryPipelineBuilder & pipeline, const Buil
 
     if (optimize_distinct_in_order)
     {
+        LOG_DEBUG(getLogger(), "Input sort description    ({}): {}", input_stream.sort_description.size(), dumpSortDescription(input_stream.sort_description));
```

Comment:
This looks too verbose, maybe `LOG_TRACE` or even better `LOG_TEST`? @devcrafter what do you think?

Reply:
Yes, thanks for feedback. Alexey mentioned it as well. See fix #39955